src/lap_stats/lap_stats.c

```diff
@@ -129,15 +129,22 @@ static void end_lap_timing(const GpsSnapshot *gpsSnapshot)
     g_lapStartTimestamp = -1;
 }
 
-static void update_distance(const GpsSnapshot *gpsSnapshot)
+static void update_distance(const GpsSnapshot *gps_ss)
 {
-    const GeoPoint prev = gpsSnapshot->previousPoint;
-    const GeoPoint curr = gpsSnapshot->sample.point;
+        const float speed_avg =
+                (gps_ss->sample.speed + gps_ss->previous_speed) / 2;
 
-    if (!isValidPoint(&prev) || !isValidPoint(&curr))
-        return;
+        if (speed_avg < 1)
```

Comment:
I wanted a low baseline here so people moving the car don't trip it up slightly.  Anything under 2mph is slower than walking.  I could be convinced into changing this to 0.5.  0 is out of the question IMHO since it would be trivial to bump the unit and get a non-zero value.  And if we ever see a speed less than 0 then I want words with our GPS chip maker.